buyer-api/src/operations/getOrderData.js

```diff
@@ -0,0 +1,48 @@
+import { parseAsync } from 'json2csv';
+import { getRawOrderData } from '../utils/wibson-lib/s3';
+
+const NOT_FOUND = {
+  code: 'getData.not_found',
```

Comment:
What about?
```suggestion
  code: 'ODER_DATA_NOT_FOUND',
```
Isn't this the format we used to respond to the mobile app?

Reply:
Mmh we have similar codes already, like 
```javascript
const ERROR_CLOSED_DATA_RESPONSE = {
  message: "Can't accept DataReponse, closed data order",
  code: 'add_data_response.closed_data_order',
};

const ERROR_ADD_DATA_RESPONSE_INVALID_NOTARY = {
  message: "Can't accept DataReponse, invalid notary",
  code: 'add_data_response.invalid_notary',
};
```

Reply:
Oh, I see, my bad. We'll need to look into this to have a uniform interface